packages/grafana-ui/src/components/IconButton/IconButton.tsx

```diff
@@ -61,27 +61,19 @@ export const IconButton = React.forwardRef<HTMLButtonElement, Props>(
     const styles = getStyles(theme, limitedIconSize, variant);
     const tooltipString = typeof tooltip === 'string' ? tooltip : '';
 
-    // When using tooltip, ref is forwarded to Tooltip component instead for https://github.com/grafana/grafana/issues/65632
-    const button = (
-      <button
-        ref={tooltip ? undefined : ref}
-        aria-label={ariaLabel || tooltipString}
-        {...restProps}
-        className={cx(styles.button, className)}
-      >
-        <Icon name={name} size={limitedIconSize} className={styles.icon} type={iconType} />
-      </button>
+    // ref is forwarded to Tooltip component instead for https://github.com/grafana/grafana/issues/65632
+    return (
+      <Tooltip ref={ref} content={tooltip} placement={tooltipPlacement}>
+        <button
+          ref={ref}
```

Comment:
The ref is being passed to both elements, according to the comment it should not be passed to the `button`